core/modules/user/user.actions.inc

```diff
@@ -13,6 +13,18 @@
  * @ingroup actions
  */
 function user_block_user_action(User $account, &$context) {
+  global $user;
+
+  // Don't block User 1, or the user performing the action.
+  if ($account->uid === '1') {
+    backdrop_set_message(t("User 1 cannot be blocked."), 'warning');
```

Comment:
https://api.backdropcms.org/php-standards#quotes

For strings within `t()` single quotes are preferred over double:

```suggestion
    backdrop_set_message(t('User 1 cannot be blocked.'), 'warning');
```